401-code-challenges/array-shift/array-shift.test.js

```diff
@@ -0,0 +1,60 @@
+'use strict'
+
+function insertShiftArray(array, insert){
+  if (!Array.isArray(array)){
+    return 'I need an array to put something into'}
```

Comment:
This is a place I would probably throw an error. The app is likely going to break later on anyway if you return a string when they were expecting an array back. So instead of the return your could put `Throw 'I need an array to put something into'`